index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+    <title>My_Portfolio</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="header-container">
+            <a href="#" class="logo">My Logo</a>
+            <nav class="nav">
+                <i class="fa fa-bars fa-2x"></i>
+            </nav>
+        </div>
+    </header>
+
+    <main class="main">
+        <section class="section-hero">
+            <div class="hero-container">
+                <h1>I’m Eugene Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
```

Comment:
- [Optional] Please consider using an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)